src/components/UiOverlay/UiOverlay.tsx

```diff
@@ -49,6 +50,7 @@ const getEditorModeMapping = (editorMode: keyof typeof EditorModeEnum) => {
 
 export const UiOverlay = () => {
   const theme = useTheme();
+  useScrollZoom();
```

Comment:
For consistency, I think this hook should move to `useInteractionManager.ts` (that's currently where everything interaction related lives).